src/Dropdown.tsx

```diff
@@ -1,8 +1,15 @@
 import * as React from 'react';
-import Trigger, { TriggerProps } from 'rc-trigger';
+import type { TriggerProps } from 'rc-trigger';
+import Trigger from 'rc-trigger';
 import classNames from 'classnames';
-import { AnimationType, AlignType, BuildInPlacements, ActionType } from 'rc-trigger/lib/interface';
+import type {
+  AnimationType,
+  AlignType,
+  BuildInPlacements,
+  ActionType,
+} from 'rc-trigger/lib/interface';
 import Placements from './placements';
+import useAccessibility from './hooks/useAccessibility';
```

Comment:
Does it belong to #182 which should not be commited here?

Reply:
Updated with the push below